src/main/java/org/spongepowered/common/mixin/api/mcp/entity/EntityMixin_API.java

```diff
@@ -682,4 +684,32 @@ public EntityArchetype createArchetype() {
         this.get(VelocityData.class).ifPresent(manipulators::add);
     }
 
+    @Override
+    public Set<String> getTags() {
```

Comment:
`Entity#getTags` returns the original set. While I don't see any vanilla code that makes use of the fact that the backing set is mutable, mods very well might. It would be stupid for them to do so, but they might.

Thankfully, there is a really easy way to sidestep this: make this method return a `Collection<String>`! While Java doesn't let you vary only by return type, no such restriction exists on the JVM. That way our method and the vanilla `Set<String> getTags()` method can co-exist peacefully.